wiki/Skinning/Interface/en.md

```diff
@@ -925,9 +924,9 @@ _`pause-overlay.png`_ (transparent; not included)
 
 **Notes:**
 
-- If one is not skinned, the default is to dim the background.
+- If one is not skinned, the background of the map will be dimmed by default.
```

Comment:
rephrase to `If not skinned, the background of the map will be dimmed by default.` or similar